README.md

```diff
@@ -54,7 +54,7 @@ We would love to hear from the larger community: please provide feedback proacti
 
 Unless otherwise noted, all published artifacts support Java 8 or higher.
 
-**Android Disclaimer:** _It's highly recommended enabling [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) in order to avoid odd Runtime exceptions (i.e. Issue [#4137](https://github.com/open-telemetry/opentelemetry-java/issues/4137))._
+**Android Disclaimer:** For compatibility reasons, [library desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring) must be enabled. (i.e. Issue [#4137](https://github.com/open-telemetry/opentelemetry-java/issues/4137)).
```

Comment:
I'm not sure referencing an issue in a README is a great idea. I'd prefer we just stick with the note, and not reference the issue, personally.